base_rest/components/service.py

```diff
@@ -80,6 +89,12 @@ def _get_schema_for_method(self, method_name):
             raise NotImplementedError(validator_method)
         return getattr(self, validator_method)()
 
+    def _get_schema_output_for_method(self, method_name):
+        validator_method = '_validator_return_%s' % method_name
```

Comment:
@acsonefho IMO we should change the naming convention for methods in change of providing the schemas. 
May be '_my_method_request_schema'  '_my_method_response_schema'. In the same time we should replace these methods by properties. `@property` @sebastienbeau What do you think about?

Reply:
@sebastienbeau  @sbidoul Do you have some advice on the naming convention to apply to the properties providing the schema to validate the request and the response?
`_(method_name)_(request|response)_schema`

```python
@property
_get_request_schema(self):
    pass

@property
_get_response_schema(self):
    pass

@property
_search_request_schema(self):
    pass

@property
_search_response_schema(self):
    pass

@property
_update_request_schema(self):
    pass

@property
_update_response_schema(self):
    pass
```
or
`_schema_(method_name)_(request|response)`

```python
@property
_schema_get_request(self):
    pass

@property
_schema_get_response(self):
    pass

@property
_schema_search_request(self):
    pass

@property
_schema_search_response(self):
    pass

@property
_schema_update_request(self):
    pass

@property
_schema_update_response(self):
    pass
```
or
???